python/pyspark/ml/classification.py

```diff
@@ -28,6 +28,7 @@
 
 
 __all__ = ['LogisticRegression', 'LogisticRegressionModel',
+           'BinaryLogisticRegressionSummary', 'BinaryLogisticRegressionTrainingSummary',
```

Comment:
LogisticRegressionSummary and LogisticRegressionTrainingSummary should probably be public too